organisaatio-ui/src/components/Sivut/Ryhmat/Ryhmat.tsx

```diff
@@ -4,45 +4,43 @@ import TyypitJaRyhmatKehys from '../TyypitJaRyhmatKehys/TyypitJaRyhmatKehys';
 import { useContext } from 'react';
 import { KoodistoContext, LanguageContext } from '../../../contexts/contexts';
 import Button from '@opetushallitus/virkailija-ui-components/Button';
-import Input from '@opetushallitus/virkailija-ui-components/Input';
-import { Icon } from '@iconify/react';
-import searchIcon from '@iconify/icons-fa-solid/search';
-import Select from '@opetushallitus/virkailija-ui-components/Select';
 import { useEffect } from 'react';
 import Axios from 'axios';
 import { useState } from 'react';
 import { Ryhma } from '../../../types/types';
 import NormaaliTaulukko from '../../Taulukot/NormaaliTaulukko';
 import Spin from '@opetushallitus/virkailija-ui-components/Spin';
+import { Column } from 'react-table';
 
 const urlPrefix = process.env.NODE_ENV === 'development' ? '/api' : '/organisaatio';
 
 const Ryhmat: React.FC = (props) => {
+    const { i18n, language } = useContext(LanguageContext);
     const { ryhmaTyypitKoodisto, kayttoRyhmatKoodisto } = useContext(KoodistoContext);
     const [ryhmat, setRyhmat] = useState<Ryhma[] | undefined>(undefined);
+
     useEffect(() => {
         async function fetch() {
             try {
                 const response = await Axios.get(`${urlPrefix}/organisaatio/v3/ryhmat?aktiivinen=true`);
                 const data = response.data;
-                console.log('data', data);
                 setRyhmat([...data]);
             } catch (error) {
                 console.error('error fetching', error);
```

Comment:
humm.. so perpetual spinner as error handling?

Reply:
error handling is not properly implemented yet. Maybe this could be done at the same time?

Reply:
yep, that the point. by isolating api access to one place, it's much more easier to implement something cross-cutting like error handling

Reply:
nämä muutettu